test/cmdlineTests/strict_asm_optimizer_steps/output

```diff
@@ -69,20 +70,18 @@ sub_0: assembly {
       0x40
         /* "strict_asm_optimizer_steps/input.yul":287:302   */
       mstore
-        /* "strict_asm_optimizer_steps/input.yul":348:349   */
-      0x04
-        /* "strict_asm_optimizer_steps/input.yul":332:346   */
+        /* "strict_asm_optimizer_steps/input.yul":533:547   */
       calldatasize
-        /* "strict_asm_optimizer_steps/input.yul":329:350   */
-      lt
-        /* "strict_asm_optimizer_steps/input.yul":322:351   */
+        /* "strict_asm_optimizer_steps/input.yul":526:548   */
       iszero
-        /* "strict_asm_optimizer_steps/input.yul":319:321   */
-      pop
-        /* "strict_asm_optimizer_steps/input.yul":570:571   */
+        /* "strict_asm_optimizer_steps/input.yul":523:524   */
       0x00
-        /* "strict_asm_optimizer_steps/input.yul":567:568   */
+        /* "strict_asm_optimizer_steps/input.yul":516:549   */
+      sstore
```

Comment:
Why does it introduce an sstore?

Reply:
Because I introduced `sstore` in the input :) See https://github.com/ethereum/solidity/pull/10286#discussion_r617462279